### PR TITLE
remove fmt.Println

### PR DESCRIPTION
**Pull request by wrfly**
replace fmt.Println() with log.Debug()

### CODE REVIEW

File: backends/etcdv3/client.go [hunk 1]
```diff
@@ -119,7 +120,7 @@ func (c *Client) WatchPrefix(prefix string, keys []string, waitIndex uint64, sto
 
 	for wresp := range rch {
 		for _, ev := range wresp.Events {
-			fmt.Println(string(ev.Kv.Key))
+			log.Debug(fmt.Sprintf("Key updated %s", string(ev.Kv.Key)))
```
Comment (okushchenko) [inline backends/etcdv3/client.go]: `Log.Debug` does Sprintf for you, so it's redundant here.
https://github.com/kelseyhightower/confd/blob/master/log/log.go#L51-L54

Comment (wrfly) [reply backends/etcdv3/client.go]: Sorry about that, I'll amend it.

Comment (wrfly) [reply backends/etcdv3/client.go]: @okushchenko done, PTAL

File: backends/etcdv3/client.go [hunk 2]
```diff
@@ -119,7 +119,7 @@ func (c *Client) WatchPrefix(prefix string, keys []string, waitIndex uint64, sto
 
 	for wresp := range rch {
 		for _, ev := range wresp.Events {
-			fmt.Println(string(ev.Kv.Key))
+			log.Debugf("Key updated %s", string(ev.Kv.Key))
```
Comment (okushchenko) [inline backends/etcdv3/client.go]: There is no such method as `Debugf`, please change in to `Debug`

Comment (wrfly) [reply backends/etcdv3/client.go]: embarrassing...

**Pull request merged**